src/components/FormField/component.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {
+  Form, Input
+} from 'antd'
+import PropTypes from 'prop-types'
+
+const FormField = ({
+  placeholder, validateStatus, help, field, prefix: PrefixComponent, ...inputProps
+}) => (
+  <Form.Item
+    validateStatus={validateStatus}
+    help={help}
```

Comment:
`help` is not clear name, i have no idea what it does. please, think about more understandable name.